lemur/constants.py

```diff
@@ -12,6 +12,9 @@
 SUCCESS_METRIC_STATUS = "success"
 FAILURE_METRIC_STATUS = "failure"
 
+# fallback if we don't have an oauth state token secret configured
+OAUTH_STATE_TOKEN_SECRET_FALLBACK = b'0auth'
```

Comment:
I am wondering, if we could just create the secret on the fly for the fall_back secret?
```
OAUTH_STATE_TOKEN_SECRET_FALLBACK = base64.b64encode(get_random_secret(32).encode('utf8'))
```

likely between deploys the state cannot be verified correctly, and if there are multiple instances of Lemur, each will have varying keys, which shouldn't be a problem, since the request will be coming back to the same instance. 


any thoughts @havron

Reply:
Makes sense to me - perhaps it should move to `utils`, then, since it's no longer a constant. Updating!

Reply:
Updated it, and pulled the `get_random_secret(length)` into `utils` as well.